lib/services/microsoft_teams.rb

```diff
@@ -205,8 +204,13 @@ def facts
 
         parsed.css('span.deleted').each { |node| node.name = 'strike' }
         parsed.css('span.inserted').each { |node| node.name = 'strong' }
-        obj["value"] = parsed.to_s
+
+        obj["value"] = html_to_markdown(parsed.to_s)
       end
     end
   end
+
+  def html?(string)
+    !!(string =~ /<[^>]*>/)
+  end
```

Comment:
Please add a note or something lol.

Like on one hand, it's self documenting. It's clear what the function intends to do.

What it is doing? Something. That's for sure.